docs/docs/user/subscribing.md

```diff
@@ -39,18 +39,19 @@ Minimal request:
 
 All options:
 
-Option                               | Description                                    | Default value
------------------------------------- | ---------------------------------------------- | -------------
-trackingEnabled                      | track incoming messages?                       | false
-subscriptionPolicy.rate              | maximum sending speed in rps (per DC)          | 400
-subscriptionPolicy.messageTtl        | inflight Time To Live in seconds               | 3600
-subscriptionPolicy.messageBackoff    | backoff time between retry attempts in millis  | 100
-subscriptionPolicy.retryClientErrors | retry on receiving 4xx status                  | false
-subscriptionPolicy.requestTimeout    | request timeout in millis                      | 1000
-subscriptionPolicy.inflightSize      | max number of pending requests                 | 100
-headers                              | additional HTTP request headers                | [] (array of headers)
-filters                              | used for skipping unwanted messages            | [] (array of filters)
-endpointAddressResolverMetadata      | additional address resolver metadata           | {} (map)
+Option                               | Description                                     | Default value
+------------------------------------ | ----------------------------------------------  | -------------
+trackingEnabled                      | track incoming messages?                        | false
```

Comment:
I'm not sure whether adding another boolean option is easy to understand. People will wonder if both have to enabled in order to track all the events, or just once. 
 
I think that I'll be nicer to have a single property, that is of enum type, and accepts multiple options:
`TRACK_ALL`, `TRACK_DISCARDED_ONLY`, `TRACKING_OFF`. 
Still we could make it possible to keep compliant with the current API, so `trackingEnabled` would map to `TRACK_ALL` etc. Enum would be just an implementation detail.
This way we'll have only one setting value.

If it's done, we can modify our web console and provide a single `<select>` instead of two checkboxes.

What do you think @adididas122 ?